core/src/main/scala/com/typesafe/tools/mima/core/Filters.scala

```diff
@@ -27,13 +29,4 @@ object ProblemFilters {
     val problemClass: Class[_ <: ProblemRef] = Class.forName("com.typesafe.tools.mima.core." + problemName).asInstanceOf[Class[_ <: ProblemRef]]
     exclude(name)(ClassTag(problemClass))
   }
-
-  private case class ExcludeByPackage(excludedPackageName: String) extends ProblemFilter {
-    def apply(problem: Problem): Boolean = {
-      // careful to avoid excluding "com.foobar" with an exclusion "com.foo"
-      !problem.matchName.getOrElse("").startsWith(excludedPackageName + ".")
-    }
-  }
-
-  def excludePackage(packageName: String): ProblemFilter = new ExcludeByPackage(packageName)
```

Comment:
This is public API - can't just remove the method.

Why isn't CI failing? Does MiMa not use MiMa in CI validation?